org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/factory/DefaultFeedbackService.java

```diff
@@ -113,10 +115,20 @@ protected void showEmailDialog(byte[] imageBytes) {
 			dialog.setAttribute(Window.MODE_KEY, Mode.OVERLAPPED);			
 			
 			MSystem system = MSystem.get(Env.getCtx());
-			if (!Util.isEmpty(system.getSupportEMail())) 
+
+			for (String s : getFeedbackRecipient(MSysConfig.FEEDBACK_EMAIL_TO).split(","))
+				if(EMail.validate(s.trim()))
```

Comment:
Should check whether s is empty since EMail.validate is kind of heavy and can product warning to server log if s is empty string.